tests/integration/test_services_auto_tagging_copilot_feedback.py

```
@@ -0,0 +1,1217 @@
+
+    def test_preview_target_not_directory(self, tmp_path: Path) -> None:
+        from file_organizer.services.copilot.rules.preview import PreviewEngine
+
+        f = tmp_path / "not_a_dir.txt"
+        f.write_text("file")
+        rule_set = self._make_rule_set()
+        engine = PreviewEngine()
+        result = engine.preview(rule_set, f)
+        assert len(result.errors) == 1
+
+    def test_preview_summary_string(self, tmp_path: Path) -> None:
```

Ruff (0.15.9)

[warning] Unused method argument: `tmp_path`

(ARG002)